plugins/modules/foreman_global_parameter.py

```diff
@@ -84,34 +99,32 @@
 
 RETURN = ''' # '''
 
-try:
-    from nailgun.entities import (
-        CommonParameter,
-    )
-
-    from ansible.module_utils.ansible_nailgun_cement import (
-        find_entities,
-        naildown_entity_state,
-        sanitize_entity_dict,
-    )
-except ImportError:
-    pass
-
-from ansible.module_utils.foreman_helper import ForemanEntityAnsibleModule
 
+from ansible.module_utils.foreman_helper import ForemanEntityApypieAnsibleModule
+import json
 
 # This is the only true source for names (and conversions thereof)
 name_map = {
     'name': 'name',
     'value': 'value',
+    'parameter_type': 'parameter_type',
 }
 
 
+def value_to_str(value, parameter_type):
```

Comment:
I think this can be defined in the helper because it can be reused for host / hosgroup parameters